test/fixtures/parser/exasol/SelectStatement.sql

```diff
@@ -51,3 +51,40 @@ ROW_NUMBER () OVER (
 ORDER BY
     col1 DESC,
     col2 DESC;
+----
+SELECT x WITH INVALID UNIQUE(myid) FROM t;
+----
+SELECT * FROM values('x', 'y');
+----
+SELECT * FROM values('x', 'y') AS x(c1,c2);
+----
+SELECT * FROM values(('x','2'), ('y','2')) AS x(c1,c2);
+----
+SELECT * FROM(VALUES 1,2,3);
+----
+SELECT * FROM(VALUES 1,2,3) AS xs(n1);
+----
+SELECT * FROM VALUES BETWEEN 1 AND 15 WITH STEP 4;
+----
+SELECT first_name,name WITH INVALID FOREIGN KEY (nr) from T1
+REFERENCING T2 (id);
+----
+SELECT * WITH INVALID FOREIGN KEY (first_name,name) from T1
+REFERENCING T2;
+----
+SELECT INVALID FOREIGN KEY (nr,first_name,name) from T1
+REFERENCING T2 (id, first_name,name);
+----
+SELECT * INTO TABLE t2 FROM t1 ORDER BY 1;
+----
+SELECT date'2021-09-21' FROM dual;
+----
+SELECT INVALID PRIMARY KEY (first_name) from T1;
+----
+SELECT  JSON_EXTRACT(json_str, '$."@id"', '$.error()') 
+        EMITS
+        (
+            id VARCHAR(2000),
+            error_column VARCHAR(2000000)
+        )
+FROM t;
```

Comment:
```suggestion
FROM t;

```

Reply:
Thanks for the review. Solved your suggestions.